packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -182,6 +182,12 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     }
   }
 
+  public componentWillMount() {
+    if (this._groupedList && this.state.isCollapsed) {
+      this._groupedList.toggleCollapseAll(this.state.isCollapsed);
+    }
+  }
```

Comment:
can't we pass `this.state.isCollapsed`  to `GroupedList` in props?

Reply:
You're right, we already pass `groupProps` (including `groupProps.isAllGroupsCollapsed`) to `<GroupedList />`, so we don't need this. Updated!